releasenotes/notes/update-prometheus-services-dd195876e162251c.yaml

```diff
@@ -3,7 +3,7 @@ upgrade:
   - |
     Update Prometheus services to latest releases:
 
-    * prometheus-alertmanager: 0.27.0 -> 0.28.0
+    * prometheus-alertmanager: 0.28.0 -> 0.28.1
```

Comment:
I don't think we do release notes on downstream forks outside of SKC. As long as the change is clearly labelled in the SKC release note then you can just remove this change.

Reply:
This is just a cherypick from upstream.

git fetch https://review.opendev.org/openstack/kolla refs/changes/39/947239/5 && git cherry-pick FETCH_HEAD